### PR TITLE
martin: 0.5.0 → 0.6.1

### PR DESCRIPTION
**Pull request by sikmir**
###### Things done
- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by raphaelr — COMMENTED** [review]
[Changelog for 0.6.0](https://github.com/maplibre/martin/releases/tag/v0.6.0)  
[Changelog for 0.6.1](https://github.com/maplibre/martin/releases/tag/v0.6.1)  
There is one breaking change, the `--watch` cli option was removed, which should be fine

The .patch file needs to be deleted

**Comment by raphaelr** [comment]
Result of `nixpkgs-review pr 202221` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>martin</li>
  </ul>
</details>

It gets far enough to complain about me not having a database.

File: pkgs/servers/geospatial/martin/default.nix [hunk 1]
```diff
@@ -2,31 +2,26 @@
...
 
   buildInputs = lib.optional stdenv.isDarwin Security;
 
   doCheck = false;
 
   meta = with lib; {
     description = "Blazing fast and lightweight PostGIS vector tiles server";
-    homepage = "https://martin.urbica.co/";
+    homepage = "https://martin.maplibre.org/";
     license = licenses.mit;
```
Comment (raphaelr) [inline pkgs/servers/geospatial/martin/default.nix:23]: The new license is "MIT OR Apache-2.0": https://github.com/maplibre/martin/pull/364/files

I'm unsure if/how to express this.

**Review by raphaelr — APPROVED** [review]

File: pkgs/servers/geospatial/martin/default.nix [hunk 2]
```diff
@@ -2,31 +2,26 @@
...
 
   meta = with lib; {
     description = "Blazing fast and lightweight PostGIS vector tiles server";
-    homepage = "https://martin.urbica.co/";
-    license = licenses.mit;
+    homepage = "https://martin.maplibre.org/";
+    license = with licenses; [ mit /* or */ asl20 ];
     maintainers = with maintainers; [ sikmir ];
-    platforms = with platforms; linux ++ darwin;
+    platforms = with platforms; unix;
```
Comment (SuperSandro2000) [inline pkgs/servers/geospatial/martin/default.nix:25]:
```suggestion
```
That is the default for buildRustPackage

**Comment by sikmir** [comment]
Result of `nixpkgs-review pr 202221` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>martin</li>
  </ul>
</details>

**Pull request merged**